### PR TITLE
Fix ingress annotation and cert-manager configuration 👼

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

This is what was required (with some crazy, not useful hack on my part
😹) to fix https on prow.tekton.dev

As follow-up, we need a little bit more docs/link to help debug potential problems :angel: 

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: prow/clusterissuer.yaml [hunk 1]
```diff
@@ -7,8 +7,8 @@ spec:
     email: dlorenc@google.com
     privateKeySecretRef:
       name: letsencrypt-prod
-    server: https://acme-staging-v02.api.letsencrypt.org/directory
```
Comment (vdemeester) [inline prow/clusterissuer.yaml]: This was problematic as the generated cert was *fake*

File: prow/ingress.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
   annotations:
-    cert-manager.io/acme-http01-edit-in-place: "true"
+    acme.cert-manager.io/acme-http01-edit-in-place: "true"
```
Comment (vdemeester) [inline prow/ingress.yaml]: This was *blocking* the actual cert generation as `cert-manager` was trying to create an additionnal ingress and thus, the HTTP challenge was always `404`

**Comment by vdemeester** [comment]
I think it really fixed it :angel: 
/meow

/cc @bobcatfish @dibyom

File: prow/ingress.yaml (hunk 2 above)
Comment (afrittoli) [reply prow/ingress.yaml]: I think this should be `acme.cert-manager.io/http01-edit-in-place: "true"` ?
https://cert-manager.io/docs/tutorials/acme/ingress/

Comment (vdemeester) [reply prow/ingress.yaml]: yes.. I forgot to push my last change :joy:

**Comment by dlorenc** [comment]
/lgtm

Whoops!

**Pull request merged**